cli/src/lib/npm/npmLibDefs.js

```diff
@@ -329,12 +329,15 @@ async function _npmExists(pkgName: string): Promise<Function> {
   return got(pkgUrl, {method: 'HEAD'});
 }
 
+export const CACHE_REPO_EXPIRY = 1000 * 60;
```

Comment:
this is already defined in 
https://github.com/flow-typed/flow-typed/blob/master/cli/src/lib/cacheRepoUtils.js#L11

Maybe export that and use it instead?